### PR TITLE
Fluid box consistency

### PR DESCRIPTION
**Pull request by TwentyEighty**
Makes fluidboxes for all relevant plant and mushroom farms as consistent as possible. Attempts were made to make the fewest factory breaks possible.

### CODE REVIEW

File: prototypes/recipes/cadaveric-arum/recipes-auto-cadaveric.lua [hunk 1]
```diff
@@ -48,7 +48,7 @@ FUN.autorecipes {
 				{
                     {name='cadaveric-arum-seeds', add_amount = 10},
                     {name='pesticide-mk01', amount = 2},
-					{name='acidgas', amount = 50},
+					{name='acidgas', amount = 50, type = "fluid", fluidbox_index = 3},
```
Comment (oorzkws) [inline prototypes/recipes/cadaveric-arum/recipes-auto-cadaveric.lua:51]: You add type=fluid here but not elsewhere, why?

Comment (TwentyEighty) [reply prototypes/recipes/cadaveric-arum/recipes-auto-cadaveric.lua]: I didn't understand how the auto recipes worked at first. Removed

**Review by kingarthur91 — APPROVED** [review]

**Comment by oorzkws** [comment]
Personally I think a fluid declaration everywhere is safest (i.e. what if someone makes an item called water?!) but I think that's out of scope.

**Comment by kingarthur91** [comment]
> Personally I think a fluid declaration everywhere is safest (i.e. what if someone makes an item called water?!) but I think that's out of scope.

Then that person should be keelhauled and they're mod marked incompatible for doing something so incredibly stupid. 

Like most things I do. It's written to work for py as is. I'm not going to spend time worrying about solutions to problems that aren't actually problems yet.

**Pull request merged**